packages/examples/packages/settings-page/README.md

```diff
@@ -0,0 +1,22 @@
+# `@metamask/settings-page-example-snap`
+
+This snap demonstrates how to use `endowment:page-settings` permission to show
```

Comment:
Perhaps we should add a note that this will initially only be available to preinstalled Snaps?